tests/unit/test_sagemaker_steps.py

```diff
@@ -275,6 +275,7 @@ def test_training_step_creation_with_placeholders(pca_estimator):
     execution_input = ExecutionInput(schema={
         'Data': str,
         'OutputPath': str,
+        'HyperParameters': str
```

Comment:
Just to be clear, it's not this line that would catch the regression, it's line 296 that assigns `hyperparameters=execution_input['HyperParameters']`. It could by any JSONPath.

Reply:
yep, tagged my comment to the wrong line but that is what i meant. exercising hyperparameters as a placeholder would have caught this regression